### DISCUSSION TITLE
/usr/bin/code-server: 28: exec: /usr/lib/node: not found

### DISCUSSION THREAD

**Discussion by cvb27** [coder/code-server#6359] [General]
I tried the suggestions at issue # 1685 but I still get the same error.

![image](https://github.com/coder/code-server/assets/111543267/10af833f-4d4b-4afd-ae69-998f21d12a58)

Can you please help me?

Thank you in advance.

**Comment by code-asher** [comment]
How are you installing code-server?  And on what platform?

**Comment by cvb27** [comment]
I downloaded the binary from:

https://github.com/coder/code-server/releases

and then moved the code-server file to /bin directory.

I'm using Linux/Ubuntu.

**Reply by code-asher** [reply]
code-server is not a single binary so all the files need to be kept together; moving the one `bin/code-server` script will not work unfortunately.

There are a few options but the best one is to install with the .deb since that will set up everything for you.

But if you need to install manually one of those two options in the screenshot should have worked.  For example, I ran this on Ubuntu:

```
$ wget https://github.com/coder/code-server/releases/download/v4.16.1/code-server-4.16.1-linux-amd64.tar.gz
$ tar -xf code-server-4.16.1-linux-amd64.tar.gz
$ sudo ln -s $PWD/code-server-4.16.1-linux-amd64/bin/code-server /usr/bin/code-server
$ code-server --version
4.16.1 94ef3776ad7bebfb5780dfc9632e04d20d5c9a6c with Code 1.80.2
```

Here I just downloaded code-server into my home directory but you might want to download code-server into `/usr/lib` or something like that.
